docs/src/mock.md

```diff
@@ -435,3 +435,122 @@ pwsh bin/Debug/netX/playwright.ps1 open --save-har=example.har --save-har-glob="
 ```
 
 Read more about [advanced networking](./network.md).
+
+## Mock WebSockets
+
+The following code will intercept WebSocket connections to `**/ws` and mock entire communcation over the WebSocket, instead of connecting to the server. This example responds to a `"request"` with a `"response"`.
+
+```js
+await page.routeWebSocket('/ws', ws => {
```

Comment:
We have not mentioned BaseURL before in this document and this paragraph requires it - I recommend adding a reference to it or relying on full URLs.